app/Http/Controllers/TaskController.php

```diff
@@ -116,16 +116,19 @@ public function spotInfo($spotId)
                         BETWEEN '.$long.' - '.$radius.'/(69 * COS(RADIANS('.$lat.')))
                         AND '.$long.' + '.$radius.'/(69 * COS(RADIANS('.$lat.'))))r
```

Comment:
What "r" means in the code below ?
 `AND '.$long.' + '.$radius.'/(69 * COS(RADIANS('.$lat.'))))r`

We should ask BlueT.